what_is.md

```diff
@@ -172,13 +174,15 @@ Here's the take-home message:
 Tree sequences are efficient because they don't store each tree separately
 ```
 
-Extending the analysis plotted at the start of this page, this efficiency is predicted
-to extend to the case of simulating chromosomes for every human on the planet. Moreover,
-the advantage of using tree sequences persists even when compared to compressed versions
-of the standard VCF storage format (original published data
-[here](https://www.nature.com/articles/s41588-019-0483-y/figures/1)). Simply by virtue
-of the reduced file size, simulated genome data stored as a tree sequence can be several
-orders of magnitude faster to process than other storage formats.
+Below is an extension of the plot at the top of this page, showing predicted
+file sizes when storing not just millions, but billions of human-like genomes:
+enough to encompass every human on the planet. This demonstrates that the tree sequence
+encoding leads to savings of many orders of magnitude, even when compared against
+compressed versions of the standard VCF storage format (original published data
+[here](https://www.nature.com/articles/s41588-019-0483-y/figures/1)). What's more,
+simply by virtue of the reduced file size, analysing these genomes can be
```

Comment:
That's over-simplifying I think. It's not just because the files sizes are smaller, it's because we don't have to decompress the data to do the computations. The compressed form is the form we operate on directly, not something that needs to be decompressed before we can do anything with it.